api/v1/controllers/RegistrationController.js

```diff
@@ -378,6 +397,8 @@ router.put('/attendee/decision/:id',  middleware.request(requests.AttendeeDecisi
 router.put('/attendee/:id(\\d+)', middleware.request(requests.AttendeeRequest),
 	middleware.permission(roles.ORGANIZERS), updateAttendeeById);
 
+router.get('/attendee/byuser/:id(\\d+)', middleware.permission(roles.HOSTS), fetchAttendeeForVolunteer);
```

Comment:
Let's just call this `/attendee/user/:id(...)`. I know that's weird and is probably why you included `by`, but I personally dislike multi-word parameters.